include/configs/piksiv3_prod.h

```diff
@@ -190,9 +190,6 @@
 
 #define CONFIG_SPL_STACK                  0xfffffe00
 
-/* Use image table */
-#define CONFIG_SPL_BOARD_LOAD_IMAGE
```

Comment:
the one logical change.
This `#define` is identical in both the TPL and SPL sections, so I moved it to the end outside the `#ifdef`